c/polyjuice.h

```diff
@@ -1150,7 +1150,7 @@ int handle_message(gw_context_t* ctx,
 
   debug_print_int("[handle_message] msg.kind", msg.kind);
   /* NOTE: msg and res are updated */
-  if (to_address_exists && code_size > 0 && (is_create(msg.kind) || msg.input_size > 0)) {
+  if (to_address_exists && code_size > 0 && (is_create(msg.kind) || msg.input_size >= 0)) {
```

Comment:
`msg.input_size` has type `size_t` which is an unsigned integer type and can not be less then `0`. So just remove the `|| msg.input_size >= 0` check.